src/main/java/org/folio/circulation/domain/ServicePoint.java

```diff
@@ -1,55 +1,55 @@
 package org.folio.circulation.domain;
 
-import io.vertx.core.json.JsonObject;
-
-import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getIntegerProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getObjectProperty;
-import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import io.vertx.core.json.JsonObject;
 
 public class ServicePoint {
   private final JsonObject representation;
-  
+
   public ServicePoint(JsonObject representation) {
     this.representation = representation;
   }
-  
-  public Boolean isPickupLocation() {
+
+  public boolean isPickupLocation() {
     return representation.getBoolean("pickupLocation");
```

Comment:
Should we use default value here? It is possible to get NPE here, when the property is undefined.
```suggestion
    return representation.getBoolean("pickupLocation", false);
```